### PR TITLE
crontab exorcism

### PR DESCRIPTION
**Pull request by dbutenhof**
PBENCH-1206

When use of crontab was removed in favor of a timed systemd service, some loose ends were overlooked. Also, the BindsTo directive was put in the wrong section of the timer service file, leading to (apparently innocuous) warnings on startup. This does some cleanup.

### CODE REVIEW

**Review by ndokos — APPROVED** [review]

File: server/lib/config/pbench-server-default.cfg [hunk 1]
```diff
@@ -68,9 +68,6 @@ workers = 3
 # Set the gunicorn worker timeout. Setting it to 0 has the effect of infinite timeouts
 worker_timeout = 9000
 
-# Default roles this pbench server takes on, see crontab roles below.
-roles = pbench-results
-
```
Comment (webbnh) [inline server/lib/config/pbench-server-default.cfg:73]: Ditto for `server/pbenchinacan/etc/pbench-server/pbench-server.cfg`?

**Review by webbnh — APPROVED** [review]
Looks good, but I think you missed a spot.

File: server/lib/config/pbench-server-default.cfg (hunk 1 above)
Comment (dbutenhof) [reply server/lib/config/pbench-server-default.cfg:73]: No: we only define the crontab roles in the *default* config file. We don't duplicate that in `pbench-server.cfg`.

Comment (webbnh) [reply server/lib/config/pbench-server-default.cfg:73]: OK.  What does [this line](https://github.com/dbutenhof/pbench/blob/f5f70532ea65cbd7c24e8b2821415691c1cf43e9/server/pbenchinacan/etc/pbench-server/pbench-server.cfg#L14) do?

Comment (dbutenhof) [reply server/lib/config/pbench-server-default.cfg:73]: It looks like it's doing the backstroke ... but it's in the wrong pond. 😦

**Pull request merged**